src/Stratis.Features.SQLiteWalletRepository/SQLiteWalletRepository.cs

```diff
@@ -1350,7 +1388,7 @@ public IEnumerable<UnspentOutputReference> GetSpendableTransactionsInAccount(Wal
                 {
                     bool isColdCoinStake = txData.IsColdCoinStake ?? false;
 
-                    // Skip listing the UTXO if this is a normal wallet, and the UTXO is marked as an cold coin stake.
+                    // Skip listing the UTXO if this is a normal wallet, and the UTXO is marked as a cold coin stake.
                     if (isColdCoinStake)
                     {
```

Comment:
Shouldn't this logic apply to watch-only wallets as well? Try to re-use the existing code as much as possible.